### PR TITLE
User can login

### PR DESCRIPTION
**Pull request by ekiira**
What does this PR do?
Closes issues #7

Description of Task to be completed?
Created a login form for book gallery users

How should this be manually tested?
running ( npm start ) then add '/login' to the url path

Screenshot 2020-07-31 at 23 32 42

### CODE REVIEW

**Comment by ekiira** [comment]
I had issues with my repo so had to delete it and fork again

**Comment by mazma1** [comment]
@ekiira that's ok.
This is the preview link: https://deploy-preview-19--booksgallery.netlify.app

Superb in SuperbBooks is still not bolder

**Comment by ekiira** [comment]
oh, i made a font weight of 900 or should i reduce the font weight of Books so that the Superb is more pronounced

**Comment by mazma1** [comment]
Sure, you could try that.

I think the mockup actually used a different font that is more bulky

File: src/styles/Login.css [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  .login {
+    width: 75%;
+  }
+}
+
+@media (min-width: 748px) and (max-width: 1000px) {
+  .login {
+    width: 55%;
+  }
+}
```
Comment (mazma1) [inline src/styles/Login.css]: Add a new line to the end of this file

File: src/components/Login.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            </div>
+      </div>
+     </div>
+    </div>
+
+    </div>
+  )
+}
+
+export default Login;
```
Comment (mazma1) [inline src/components/Login.js]: Add a new line to the end of this file

**Comment by mazma1** [comment]
Looks like reducing the font weight didn't quite work.

I'll merge this and that can be fixed in another issue. You can add the fix if you still feel up to it. Well done! 💪🏽

**Review by mazma1 — APPROVED** [review]

**Comment by ekiira** [comment]
Thank You, can we still take up issues or it has ended

**Comment by mazma1** [comment]
I think he challenge has ended, but the project is still open for contributions

**Pull request merged**